edenia/antelope-tools
# Fix/accounts and endpoints UI bugs 1233

### Fix UI bugs in Endpoints and Account and Contracts page 

### What does this PR do?

- Resolve #1233
- Resolve #1231
- Add the second link to the block of the TPS All Time High

### Steps to test

1. Run the project locally
2. Go to Accounts and Contracts page
3. Check the Contract Tables section
4. Go to the endpoints page
5. Check that the url aren't overlapping
6. Go to main page
7. Check the links with both blocks of the ATH 

### Screenshots

## Accounts and Contracts

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/665e1b51-312f-46a8-a10d-a20cb37f74dd)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/e34da0d7-318c-4b68-8ac8-4803bc909b49)

## Endpoints Table

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/eab5b179-fa74-43e6-a6da-65dc26bec677)

## Main page

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/331ddb05-7f74-44ed-9582-baac686623b9)
